client/src/modules/list/components/ListItem.jsx

```diff
@@ -28,38 +125,54 @@ class ListItem extends PureComponent {
       voteForItem,
       votesCount
     } = this.props;
-    const { done } = this.state;
+    const { done, areDetailsVisible } = this.state;
     return (
       <li
-        className={classNames('items-list__item', {
-          'items-list__item--restore': !done && archived,
-          'items-list__item--done': done || archived
+        className={classNames('list-item', {
+          'list-item--restore': !done && archived,
+          'list-item--done': done || archived,
+          'list-item--details-visible': areDetailsVisible
         })}
       >
-        <input
-          className="item-list__input"
-          id={`option${id}`}
-          name={`option${id}`}
-          type="checkbox"
-        />
-        <label
-          className="items-list__label"
-          htmlFor={`option${id}`}
-          id={`option${id}`}
-          onClick={this.handleItemToggling(authorName, id, archived)}
+        <div
+          className="list-item__top"
+          onMouseEnter={this.handleMouseEnter}
+          onMouseLeave={this.handleMouseLeave}
         >
-          <span className="items-list__data">
-            <span>{name}</span>
-            <span className="items-list__author">{`Added by: ${authorName}`}</span>
-          </span>
-          {!archived && (
-            <VotingBox
-              voteForItem={voteForItem}
-              votesCount={votesCount}
-              isVoted={isVoted}
-            />
-          )}
-        </label>
+          {this.renderChevron()}
+          <input
```

Comment:
there is no visual indicator that item (and voting button) is focused...

Reply:
Do you mean outline??

Reply:
outline or other visual styles to show the user which element has focus now

Reply:
It have different border color on focus